hack/docgen/main.go

```diff
@@ -304,8 +313,14 @@ func printAPIDocs(paths []string, sectionLink string) {
 		fmt.Println("| Field | Description | Scheme | Required |")
 		fmt.Println("| ----- | ----------- | ------ | -------- |")
 		fields := t[1:]
+
 		for _, f := range fields {
-			fmt.Println("|", f.Name, "|", f.Doc, "|", f.Type, "|", f.Mandatory, "|")
+			scheme := f.Type
+			// if !strings.HasPrefix(scheme, "[") {
```

Comment:
do we need this commented piece of code?

Reply:
nope, sorry - fixed